### DISCUSSION TITLE
PnP Provisioning doesn't include updated NewForm or DispForm cofigurations?

### DISCUSSION THREAD

**Discussion by PeterBehler** [pnp/powershell#4642] [Q&A]
I am deep with creating a custom pnp provisioning template file by editing the xml code and adding or removing specific attributes to be included with the template for other sites. I've learned how to add folders and files and list items, tweak the content types and site columns and site collection applications during the provisioning process. Freaking awesome for providing a tool that allows this.  

I'm having issues copying the updated New/Edit/Display forms from my source site. When I go to my site/Lists/ActionItems and use the OTB New Form, it allows me to add/remove properties from that panel. Or I can update using JSON formatting but I'm not doing that in this scenario.  I just hid a few properties.

Then I use get-pnpsitetemplate or export-pnplisttositetemplate, the xml file doesn't include any indications of the tailored NewForm.aspx. Is there a different method I need to use to include it?

I also attempted to use the get-pnpfile to export the NewForm.aspx file, I manually use add-pnpfile and added the file to the list, it didn't seem to help. So maybe that file isn't what SPO is referencing? 
I even tried using a different name and had the pnptemplate.xml reference the new path as the DefaultNewFormURL but I got the error "Unable to find an SPForm matching URL /sites/prj00000/Lists/ai/DispForm-ai.aspx"

Any pointers are appreciated. I feel like I managed to make this work once in the past. Maybe it was when I was exporting a site via SPO Powershell commands to generate a site script.

**Comment by waaromikniet** [comment]
I think In modern view the forms are not the old New/Edit/Display forms
